src/page/page3.html

```diff
@@ -1,95 +1,111 @@
+      }
 
-    .main-content {
-      height: unset;
-      padding: unset;
-    }
+      .main-content {
+        height: unset;
+        padding: unset;
+      }
 
-    .main-form legend {
-      margin-bottom: 0;
-      font-size: 16px;
-      display: none;
-      padding: 30px;
-    }
+      .error-message {
+        display: none;
+        margin-bottom: 5px;
+      }
 
-    .error-message {
-      display: none;
-      margin-bottom: 5px;
-    }
+      .show-error-message {
+        display: flex;
+      }
 
-    .show-error-message {
-      display: flex;
-    }
+      .error-icon {
+        height: 15px;
+        width: 15px;
+      }
 
-    .error-icon {
-      height: 15px;
-      width: 15px;
-    }
+      .info-message {
+        color: #f2f2f266;
+        margin: 30px;
+      }
+    </style>
+  </head>
 
-    .info-message {
-      color: #f2f2f266;
-      margin: 30px;
-    }
-  </style>
-</head>
-
-<body>
-  <div class="container">
-    <div class="content">
-      <header class="main-header">
-        <div class="main-header__container">
-          <div class="main-header__title">
-            <h1 class="title">You're done!</h1>
-            <p class="subtitle">
-              Tap Ok to finish
+  <body>
+    <div class="container">
+      <div class="content">
+        <header class="main-header">
+          <div class="main-header__container">
+            <div class="main-header__title">
+              <h1 class="title">You're done!</h1>
+              <p class="subtitle">Tap Ok to finish</p>
+            </div>
+          </div>
+        </header>
+        <section class="main-content">
+          <form
+            action="/finished"
+            method="GET"
+            id="main-form"
+            class="main-form"
+          >
+            <input style="display: none" name="next" value="close" />
+            <p class="error-message info-message show-error-message">
+              <img src="img/icon-info.svg" alt="" class="error-icon" />
+              You can reconfigure this plugin by disconnecting and reconnecting.
+              Your time statistics are safe.
+            </p>
+            <p class="error-message info-message" id="multimcfinished">
+              <img src="img/icon-info.svg" alt="" class="error-icon" />
+              If you have normal/vanilla Minecraft installed, you must double
+              click 'play' to open MultiMC.
             </p>
+          </form>
+        </section>
+        <footer class="main-footer">
+          <div class="main-footer__container">
+            <button
+              class="basic-button main-footer__button main-footer__button--primary"
+              type="submit"
+              form="main-form"
+            >
+              Ok
+            </button>
+            <a
+              class="forgot-password open"
+              href="https://github.com/TouwaStar/Galaxy_Plugin_Minecraft"
+            >
+              Integration Info
+            </a>
```

Comment:
I would just treat it the same as OK button. Do not reopen (NextStep) if user clicked "More Info"

Reply:
not obligatory

Reply:
good idea, thanks.